aries_cloudagent/anoncreds/anoncreds/routes.py

```diff
@@ -0,0 +1,434 @@
+"""Anoncreds admin routes."""
+import logging
+
+from aiohttp import web
+from aiohttp_apispec import (
+    docs,
+    match_info_schema,
+    querystring_schema,
+    request_schema,
+    response_schema,
+)
+from marshmallow import fields
+
+# from ...admin.request_context import AdminRequestContext
+from ...messaging.models.openapi import OpenAPISchema
+from ...messaging.valid import (
+    GENERIC_DID,
+    INDY_CRED_DEF_ID,
+    INDY_SCHEMA_ID,
+    INDY_VERSION,
+    UUIDFour,
+)
+
+LOGGER = logging.getLogger(__name__)
+
+SPEC_URI = ""
+
+schemaId = fields.Str(
```

Comment:
For the sake of consistency with the `...messaging.valid` field definition helpers, I recommend having these be "constants" that are dictionaries of keyword arguments to the `fields` methods.